test/test_controller.js

```diff
@@ -385,130 +258,64 @@ contract('ICO_controller', async function (accounts) {
-            });
-    });
-
-});
-
-contract("Controller ICO Bounty Pull", async function (accounts) {
-    it('test sendPullTokensTo', async function () {
+contract("ICO Controller Airdrop", async function (accounts) {
+    it('test sendAirdrop', async function () {
         let controller = await Controller.deployed();
         let token = await Token.at(await controller.token.call());
-        let pullAccount = accounts[6];
+        let airdropAccounts = [accounts[6], accounts[7]];
         try {
-            await controller.sendPullTokensTo(pullAccount, 100, {from: pullAccount});
+            await controller.sendAirdrop(airdropAccounts, [100, 110], {from: airdropAccounts[0]});
             assert.ifError('Error, only owner should be able to use the function');
         } catch (err) {
             assert.equal(err, 'Error: VM Exception while processing transaction: revert',
                 "Only owner should be able to use the function");
         }
         try {
-            await controller.sendPullTokensTo(pullAccount, web3.toWei(4000001, 'ether'));
-            assert.ifError('Error, too big amount of tokens to send allowed');
+            await controller.sendAirdrop(airdropAccounts, [web3.toWei(5000000, 'ether'), 1]);
+            assert.ifError('Error, too big amount of tokens to send is allowed');
         } catch (err) {
             assert.equal(err, 'Error: VM Exception while processing transaction: revert',
-                "It is shouldn't be able to transfer more tokens then pull supply");
+                "It is shouldn't be able to transfer more tokens then airdrop supply");
         }
-        let expectedTokenAmount = web3.toWei(3999999, 'ether');
-        await controller.sendPullTokensTo(pullAccount, expectedTokenAmount);
-        let actualTokenAmount = BigNumber(await token.balanceOf(pullAccount));
-        assert.isTrue(actualTokenAmount.isEqualTo(actualTokenAmount), "Wrong token balance");
+        let expectedTokenAmountAccount1 = web3.toWei(4999999, 'ether');
+        let expectedTokenAmountAccount2 = 100;
+        await controller.sendAirdrop(airdropAccounts, [expectedTokenAmountAccount1, expectedTokenAmountAccount2]);
+
+        let actualTokenAmount1 = BigNumber(await token.balanceOf(airdropAccounts[0]));
+        assert.isTrue(actualTokenAmount1.isEqualTo(expectedTokenAmountAccount1), "Wrong token balance account1");
+        let actualTokenAmount2 = BigNumber(await token.balanceOf(airdropAccounts[1]));
+        assert.isTrue(actualTokenAmount2.isEqualTo(expectedTokenAmountAccount2), "Wrong token balance account2");
     });
 
     it("test sender's whitelist", async function () {
         let controller = await Controller.deployed();
         let token = await Token.at(await controller.token.call());
-        let pullAccount = accounts[6];
-        let receiverAccount = accounts[7];
+        let airdropAccount = [accounts[6]];
```

Comment:
Not critical, but I think it would be clearer to have `airdropAccount` as non-array variable (same as `receiverAccount`) - this way the variable name (which hints to the single value) would match to the content.